MANIFEST.in

```diff
@@ -4,3 +4,8 @@ include gimel/VERSION
 include gimel/config.json.template
 include requirements.txt
 recursive-include gimel/vendor *
+
+exclude tox.ini
+exclude requirements.in
+exclude requirements.txt
```

Comment:
Also not totally sure. I think it depends from project to project.
FYI: [install_requires vs Requirements files](http://python-packaging-user-guide.readthedocs.io/en/latest/requirements/?highlight=requirement#install-requires-vs-requirements-files)